### ISSUE TITLE
PFS skips block when processing ChannelOpened event

### ISSUE THREAD

**Issue by karlb** [raiden-network/raiden#5443]
We see log entries like
```json
{"from_block":1765747,"to_block":1765747,"getting":0.83,"processing":0.14,"total_duration":0.97,"event_counts":{"ReceiveChannelOpenedEvent":1,"UpdatedHeadBlockEvent":1},"event":"Processed blocks","level":"info","logger":"pathfinding_service.service","timestamp":"2019-12-05 00:35:39.572985"}
{"from_block":1765749,"end_block":1765749,"event":"Querying new block(s)","level":"info","logger":"raiden_libs.blockchain","timestamp":"2019-12-05 00:36:07.707914"}
```
which shows that block 1765748 has never been processed.

**Comment by karlb** [comment]
It does not happen after most ChannelOpened events, and it does not only happen in that case:
```
{"f":1766262,"t":1766262,"counts":{"UpdatedHeadBlockEvent":1}}
{"f":1766265,"t":1766265,"counts":{"ReceiveNonClosingBalanceProofUpdatedEvent":1,"UpdatedHeadBlockEvent":1}}
```

**Comment by karlb** [comment]
After @hackaugusto mentioned that it can only be caused by some other process accessing the same db, we checked file access with `lsof` and found two containers accessing the db. When we changed the ids of the docker-compose containers, the old containers went invisible to docker-compose because they did not match the service ids given in the docker-compose.yaml, but they kept running.

I've stopped the old containers now and this should not happen again.
